pavelib/utils/test/suites/nose_suite.py

```diff
@@ -190,7 +190,8 @@ def _default_test_id(self):
 
         if self.root == 'lms':
             default_test_id += " {system}/tests.py"
-            default_test_id += " openedx/core/djangolib"
+            default_test_id += " openedx/core/djangolib/*"
+            default_test_id += " openedx/features/*"
```

Comment:
@andy-armstrong We are currently getting a test run error with the following:
```
Unable to load tests from file /home/jenkins/workspace/edx-platform-test-subset/openedx/features/README.rst
```
Is there a reason why you needed to change from `openedx/core/djangolib` to `openedx/core/djangolib/*`, or should I just try dropping `/*` from both?  I'm not sure if the issue has to do with this or something else, but just curious about this change.